service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -36,6 +48,23 @@ default void bindServiceInfo(ServiceInfo serviceInfo) {
         binder.bind(JavadslServicesRouter.class);
     }
 
+    /**
+     * Binds Service interfaces with their implementations and registers them for publishing. This method must be
+     * invoked exactly one per Lagom service unless you are developing a consume-only service (see
+     * {@link ServiceGuiceSupport#bindServiceInfo} for consume-only service support).
+     * <p>
+     * Inspects all bindings and creates routes to serve every call described in the services bound.
+     * <p>
+     * The service bindings are used to build a {@link ServiceInfo} but only the <code>locatable</code>
```

Comment:
Maybe: "The service bindings build ServiceInfo metadata for locatable services."